packages/triple-header/src/triple-header.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Meta } from '@storybook/react'
+
+import TripleHeader from './triple-header'
+import SAMPLE from './mocks/triple-header.sample.json'
+
+export default {
+  title: 'triple-header / TripleHeader',
+} as Meta
```

Comment:
```suggestion
} as ComponentMeta<typeof TripleHeader>
```
https://github.com/titicacadev/triple-frontend/blob/main/packages/action-sheet/src/action-sheet.stories.tsx#L22-L24
https://github.com/titicacadev/triple-frontend/blob/main/packages/action-sheet/src/action-sheet.stories.tsx#L122

12버전되면서 구체적인 타입을 부여하도록 변경되어서 이런식으로 적용해주시면 좋을것 같아요~